tests/management/workspace/test_view.py

```diff
@@ -148,6 +150,7 @@ def test_create_workspace_assign_parent_id(self):
         self.assertEquals(data.get("type"), "standard")
         self.assertEqual(response.get("content-type"), "application/json")
 
+    @unittest.skip('v2response_error_from_errors causes this to fail generically with "This field is required"')
```

Comment:
I think we may need to either explicitly raise this still to have a valid error message, leave it like this, or update the exception handler. If we choose anything but the latter, we could address this when we address exception handling.

Reply:
Per conversation with Alec, we'll just update the assertions here and revisit when addressing the json+problem exception handling.